tests/test_fusion.py

```diff
@@ -0,0 +1,47 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import filecmp
+import os
+from pyserini.trectools import TrecRun
+import unittest
+
+
+class TestSearch(unittest.TestCase):
+    def setUp(self):
+        self.qruns = ['tests/resources/covidex.t5', 'tests/resources/covidex.sim']
+        self.output_path = 'tests/resources/output.txt'
+
+    def test_reciprocal_rank_fusion(self):
```

Comment:
I don't like checking large run files into the repo... bloats the repo, which I'd like to keep small.

I actually like your previous testing strategy better. How about we take this:
https://github.com/castorini/anserini/blob/master/docs/experiments-covid.md#round-2

Fuse rows 1, 3, 5 and make sure we get row 7?
Fuse rows 2, 4, 6 and make sure we get row 8?

I will ensure over the long term that the DropBox links are stable.